renderer.js

```diff
@@ -121,8 +134,28 @@ document.addEventListener("DOMContentLoaded", function () {
     initialView: "dayGridMonth",
     dateClick: function (info) {
       // Use the load dates for the current and day after the current day here
-      alert("Clicked on: " + info.dateStr);
+      monthNumber = info.dateStr[5] + info.dateStr[6];
+      dayNumber = info.dateStr[8] + info.dateStr[9];
+      loadCurrentDayTasks(dayNumber, monthNumber);
     },
   });
   calendar.render();
 });
+function loadCurrentDayTasks(dayNumber, monthNumber) {
+  const existingTasks = JSON.parse(localStorage.getItem("tasks")) || [];
+
+  // Clears the tasklist
+  tasksList.innerHTML = "";
+
+  existingTasks.forEach((task) => {
+    if (
+      task.whenTask.substring(5, 7) !== monthNumber ||
```

Comment:
The only problem I encountered was this if statement because I originally used "&&" operator and wondered why it didn't work until I asked good old chatGPT.